tests/model_monitoring/db/tsdb/tdengine/test_tdengine_connector.py

```diff
@@ -80,9 +80,7 @@ def test_write_application_event(
         "result_extra_data": """{"question": "Who wrote 'To Kill a Mockingbird'?"}""",
         "result_value": result_value,
     }
-    with pytest.raises(
-        taoswswrap.tdengine_connection.TDEngineError, match="Database not exist"
-    ):
+    with pytest.raises(TDEngineError, match="Failed to run statement `USE"):
```

Comment:
Why did the error message change?

Reply:
Yup.

Reply:
> Yup.

I don't understand this response...

Reply:
Sorry, misread the question. With the subprocess mechanism removed, the exception now looks like this:
```
>                   self._conn.execute(statement)
E                   taosws.QueryError: [0x0388] Internal error: `Database not exist`
...
../../../../../mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connection.py:195: QueryError

The above exception was the direct cause of the following exception:
...
>                   raise TDEngineError(f"Failed to run statement `{statement}`") from e
E                   mlrun.model_monitoring.db.tsdb.tdengine.tdengine_connection.TDEngineError: Failed to run statement `USE mlrun_model_monitoring_`

So both the error and the statement that failed are there, which is more informative.